src/odb/test/cpp_tests.tcl

```diff
@@ -2,9 +2,9 @@ source "helpers.tcl"
 
 set test_dir [pwd]
 set openroad_dir [file dirname [file dirname [file dirname $test_dir]]]
-set tests_path [file join $openroad_dir "build" "src" "odb" "test" "cpp"]
+set tests_path [file join $openroad_dir "build" "src" "odb" "test" "cpp" ""]
```

Comment:
why the extra ""?

Reply:
It may have been there from my previous attempt. I removed it.

Thanks Matt for catching this!